### PR TITLE
FAQ custom genomes 

### PR DESCRIPTION
**Pull request by Nurzhamalyrys**
Hi everyone! 
I am moving FAQ custom genomes .

### CODE REVIEW

File: faqs/galaxy/datasets_custom_genomes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: What is a "Custom Reference Genome" ?
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_custom_genomes.md:2]: the content of the FAQ is addressing the definition (what) and the how, maybe we can rephrase the title?

File: faqs/galaxy/datasets_custom_genomes.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+- A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for single species. It is representative of a specific genome build or release.
+
+- In Galaxy, a custom reference genome is a FASTA formatted dataset that can be used in place of a native reference genome with most tools.
+
+ - custom: a dataset from the history loaded by users
+ - native: local or cached by administrators
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_custom_genomes.md:14]: could we summarise this in a short paragraph?

File: faqs/galaxy/datasets_custom_genomes.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for single species. It is representative of a specific genome build or release.
+
+- In Galaxy, a custom reference genome is a FASTA formatted dataset that can be used in place of a native reference genome with most tools.
+
+ - custom: a dataset from the history loaded by users
+ - native: local or cached by administrators 
+
+
+There are five basic steps to using a Custom Reference Genome:
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_custom_genomes.md:17]: I think that this is where the actual FAQ starts, what do you think @Nurzhamalyrys?

Comment (Nurzhamalyrys) [reply faqs/galaxy/datasets_custom_genomes.md:14]: @beatrizserrano Hi, I think , yes, main part starts here. However, it is good for a reader to know  what  a custom genome is. How do you think?

File: faqs/galaxy/datasets_custom_genomes.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+title: How to use Custom Reference Genome?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+
+- In Galaxy, a custom reference genome, which is a dataset from the history loaded by users, is a FASTA formatted dataset. It can be used in place of a native reference genome, that is local or cached by administrators, with most tools.
+- A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for single species. It is representative of a specific genome build or release.
```
Comment (beatrizserrano) [inline faqs/galaxy/datasets_custom_genomes.md:11]: This is a bit confusing to me. What about something like: 
> A reference genome is...
> You have two options to use reference genomes in Galaxy: _native_ (provided by the server administrators and used by most of the tools) and _custom_ (uploaded by users in FASTA format). This FAQ covers the latter.

I'm not an expert on this, so don't take it too seriously :)

Comment (Nurzhamalyrys) [reply faqs/galaxy/datasets_custom_genomes.md:11]: > This is a bit confusing to me. What about something like:
> 
> > A reference genome is...
> > You have two options to use reference genomes in Galaxy: _native_ (provided by the server administrators and used by most of the tools) and _custom_ (uploaded by users in FASTA format). This FAQ covers the latter.
> 
> I'm not an expert on this, so don't take it too seriously :)

Hi! 
Yeah, your version is easier to understand, so changed to your version. Please, let me know if something should be changed also.

**Review by beatrizserrano — APPROVED** [review]

**Review by annefou — APPROVED** [review]

**Pull request merged**